src/apis/study-room/exit-study-room.ts

```diff
@@ -0,0 +1,10 @@
+import { http } from '../api';
+
+type ExitStudyRoomParam = {
```

Comment:
create-study-room.ts 파일에서는 복수형으로 표현해주고 있는데요~
실제로도 여러 값을 받으니까 여기서도 복수형으로 통일해줘도 괜찮을 것 같네요!
```suggestion
type ExitStudyRoomParams = {
```